src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx

```diff
@@ -148,25 +148,25 @@ All branches that follow the `daily-release/mm-dd-yy-morning` pattern are [prote
 
 ## Github labels [#labels]
 
-Every issue needs these labels:
-- Always add `content`
-- Add one of the `pg_*` labels (see [this internal doc](https://docs.google.com/document/d/1RMLuZkF11tKf48cZttVXyEktNanZL-lZmzjL5oXphUs/edit#heading=h.9h8a1bxy0fzw) )
```

Comment:
Is this page publicly visible? I see a couple of internal links and this content all seems internal facing so was just curious...

Reply:
@stumblefiend It's a good catch. It is publicly available. In other places in the style guide I think we've instead named the internal doc and where to find it, rather than linking. For example:

```For full definitions, see the "Doc Jira and GitHub fields" doc in the internal team Google Drive.```

Thoughts on that approach @stumblefiend @bradleycamacho ?

Reply:
Does it matter if we link to a doc external folks don't have access to? It just seems like a bit of a pain to have to go look that doc up.

Reply:
But if it's necessary for security reasons or so, I'm all for it.